docs/output.md

```diff
@@ -2,40 +2,325 @@
+#├── ensembleamppred/
+├── fargene/
+├── hamronizer/
+├── hmmsearch/
+├── macrel/
+├── multiqc/
+#├── neubi/
+├── pipeline_info/
+├── prokka/
+#└── rgi/
+work/
+``` 
+-->
+```bash
+outdir/
+├── antismash/
+├── deeparg/
+├── fargene/
+├── hamronizer/
+├── hmmsearch/
+├── macrel/
+├── multiqc/
+├── pipeline_info/
+└── prokka/
+work/
+```
 
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes data using the following steps:
 
-* [FastQC](#fastqc) - Raw read QC
+* [DeepARG](#deeparg) - antimicrobial resistance gene detection, using a deep learning model
+* [fARGene](#fargene) - antimicrobial resistance gene detection, using a deep learning model
+<!--* [rgi](#rgi) - antimicrobial resistance gene detection, based on alignment to the CARD database-->
+* [hAMRonization](#hamronization) - summarizing resistance gene output from various detection tools
+* [Prokka](#prokka) - (default) open reading frame and functional protein annotation
+<!--* [prodigal](#prodigal) - (optional: replaces prokka) open reading frame annotation-->
+<!--* [acep](#acep) - antimicrobial peptide detection-->
+<!--* [ai4amp](#ai4amp) - antimicrobial peptide detection-->
+<!--* [ampir](#ampir) - antimicrobial peptide detection-->
+<!--* [amplify](#amplify) - antimicrobial peptide detection-->
+<!--* [EnsembleAMPPred](#ensembleamppred) - antimicrobial peptide detection-->
+* [Macrel](#macrel) - antimicrobial peptide detection
+<!--* [neubi](#neubi) - antimicrobial peptide detection-->
+<!--* [combiamp](#combiamp) - summarizes antimicrobial peptide detection output-->
+* [hmmsearch](#hmmsearch) - biosynthetic gene cluster detection
+* [antiSMASH](#antismash) - biosythedic gene cluster detection
 * [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
 * [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
 
-### FastQC
+### DeepARG
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `deeparg/`
+    * `db/`: contains diamond, data, database and model information
+    * `predict/`:
+        * `*.align.daa*`: Diamond alignment output.
+        * `*.align.daa.tsv`: Diamond alignment output as .tsv.
+        * `*.mapping.ARG`: contains the sequences with a probability >= --prob (0.8 default).
+        * `*.mapping.potential.ARG`: contains the sequences with a probability < --prob (0.8 default).
+
+</details>
+
+[deepARG](https://bitbucket.org/gusphdproj/deeparg-ss/src/master/) uses deep learning to characterize and annotate antibiotic resistance genes in metagenomes. It is composed of two models for two types of input: short sequence reads and gene-like sequences. In this pipeline we use the `ls` model, which is suitable for annotating full sequence genes and to discover novel antibiotic resistance genes from assembled samples. The tool `Diamond` is used as an aligner. 
+
+The `*.ARG` output files contain the following fields:
+> * ARG_NAME
```

Comment:
```suggestion

> * ARG_NAME
```

Is there a reason why you've put the list in a block quote and not just a standard bullet list?